whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/transform/FilterAuthFunction.java

```diff
@@ -72,12 +84,15 @@ public FilterAuthFunction() {
     @Override @Nonnull
     public RpslObject apply(final RpslObject rpslObject) {
         final List<RpslAttribute> authAttributes = rpslObject.findAttributes(AttributeType.AUTH);
-        if (authAttributes.isEmpty()) {
+        if (authAttributes.isEmpty() && override == null) {
             return rpslObject;
         }
 
         final Map<RpslAttribute, RpslAttribute> replace = Maps.newHashMap();
-        final boolean authenticated = isMntnerAuthenticated(rpslObject);
+        final boolean authenticated = isMntnerAuthenticated(rpslObject) ||
```

Comment:
Can you move override check into `isMntnerAuthenticated` ?

Reply:
they are different.
override doesn't have nothing to do with the mntner. isMntnerAUthenticated is checking the auth attribute of the mntner 
If I do that I should change the name of that method

Reply:
Can you create a separate method for override such as `isOverrideAuthenticated()` and move the logic into that? So it's clearer to read